Description: fix build with glibc 2.30
 glibc 2.30 release notes at
 https://savannah.gnu.org/forum/forum.php?forum_id=9515 states:
 """
 Support for the "inet6" option in /etc/resolv.conf and the RES_USE_INET6
 resolver flag (deprecated in glibc 2.25) have been removed.
 ...
 The obsolete RES_INSECURE1 and RES_INSECURE2 option flags for the DNS stub
 resolver have been removed from <resolv.h>.
 """
 And RES_AAONLY and RES_PRIMARY are already flagged as deprecated and are
 being guarded with the same fix.
Origin: upstream, https://github.com/vdukhovni/postfix/commit/3274c3cea9d739f86e84b65664aabb692e37e83f#diff-777bfb681a1cd539ddc8e1e606959ffa
Bug: http://postfix.1071664.n5.nabble.com/build-failure-with-glibc-2-30-td102511.html
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/postfix/+bug/1842923
Last-Update: 2019-09-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/postfix/src/dns/dns_str_resflags.c b/postfix/src/dns/dns_str_resflags.c
index 5f2cce5e..472394c3 100644
--- a/src/dns/dns_str_resflags.c
+++ b/src/dns/dns_str_resflags.c
@@ -52,18 +52,28 @@
 static const LONG_NAME_MASK resflag_table[] = {
     "RES_INIT", RES_INIT,
     "RES_DEBUG", RES_DEBUG,
+#ifdef RES_AAONLY
     "RES_AAONLY", RES_AAONLY,
+#endif
     "RES_USEVC", RES_USEVC,
+#ifdef RES_PRIMARY
     "RES_PRIMARY", RES_PRIMARY,
+#endif
     "RES_IGNTC", RES_IGNTC,
     "RES_RECURSE", RES_RECURSE,
     "RES_DEFNAMES", RES_DEFNAMES,
     "RES_STAYOPEN", RES_STAYOPEN,
     "RES_DNSRCH", RES_DNSRCH,
+#ifdef RES_INSECURE1
     "RES_INSECURE1", RES_INSECURE1,
+#endif
+#ifdef RES_INSECURE2
     "RES_INSECURE2", RES_INSECURE2,
+#endif
     "RES_NOALIASES", RES_NOALIASES,
+#ifdef RES_USE_INET6
     "RES_USE_INET6", RES_USE_INET6,
+#endif
 #ifdef RES_ROTATE
     "RES_ROTATE", RES_ROTATE,
 #endif
